contracts/stork_chainlink_adapter/contracts/IStorkTemporalNumericValueUnsafeGetter.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.24 <0.9.0;
+
+import "./StorkStructs.sol";
+
+
+interface IStorkTemporalNumericValueUnsafeGetter {
```

Comment:
Rather than moving `Stork.sol` into our npm package, define an interface here which has the only function we need from `Stork.sol`.